src/test/EigenPod.t.sol

```diff
@@ -1488,8 +1487,8 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         address recipient,
         uint64 withdrawalAmountGwei,
         ValidatorInfo memory validatorInfo
-    ) public {
-        _processFullWithdrawal(
+    ) public returns(IEigenPod.VerifiedWithdrawal memory) {
+        return _processFullWithdrawal(
```

Comment:
why this change? seems sensible + presumably better for integrations, but noting that this changes the interface.

Reply:
nvm, test file lol

Reply:
haha yeah just to test the VerifiedWithdrawal being correct